src/clj/puppetlabs/services/jruby/jruby_schemas.clj

```diff
@@ -72,35 +72,34 @@
                        (ifn? (:shutdown-on-error state))))))))
 
 (def PoolState
-  "A map that describes all attributes of a particular JRubyPuppet pool."
+  "A map that describes all attributes of a particular JRuby pool."
   {:pool         pool-queue-type
    :size schema/Int})
 
 (def PoolStateContainer
-  "An atom containing the current state of all of the JRubyPuppet pool."
+  "An atom containing the current state of all of the JRuby pool."
   (schema/pred #(and (instance? Atom %)
                      (nil? (schema/check PoolState @%)))
                'PoolStateContainer))
 
 (def PoolContext
-  "The data structure that stores all JRubyPuppet pools and the original configuration."
-  {:config                JRubyPuppetConfig
+  "The data structure that stores all JRuby pools and the original configuration."
+  {:config                JRubyConfig
    :pool-agent            JRubyPoolAgent
    :flush-instance-agent  JRubyPoolAgent
    :pool-state            PoolStateContainer})
 
 (def JRubyInstanceState
-  "State metadata for an individual JRubyPuppet instance"
+  "State metadata for an individual JRuby instance"
```

Comment:
I meant to make a comment about this on the PR, but I completely forgot. There were a lot of references to `JRubyPuppet` in comments and docstrings, and I think I mostly just renamed them all to `JRuby`. But I can go back and change things that were `JRubyPuppet instance` to `JRubyInstance` instead?